src/librustc_codegen_ironox/x86_asm_printer.rs

```diff
@@ -0,0 +1,79 @@
+use super::ModuleIronOx;
+
+use context::CodegenCx;
+use value::{Value, Instruction};
+
+use rustc::mir::mono::Stats;
+use rustc::util::nodemap::FxHashMap;
+use rustc_codegen_ssa::traits::MiscMethods;
+use std::cell::{Cell, RefCell};
+
+/// A convenience macro for appending instructions to a string.
+macro_rules! asm {
```

Comment:
This macro doesn't seem to be used in this PR?

Reply:
It's not, I was just trying to make sure my next PR won't be too large. I can remove it if you want.

Reply:
In general (and there are exceptions), it's better not to include unused code. So deferring it is probably the right thing to do.

Reply:
Ok.